image/lxd_test.go

```diff
@@ -14,14 +14,16 @@ import (
 	"github.com/lxc/distrobuilder/shared"
 )
 
-var lxdImageDef = shared.DefinitionImage{
-	Description:  "{{ image. Distribution|capfirst }} {{ image.Release }}",
-	Distribution: "ubuntu",
-	Release:      "17.10",
-	Architecture: "amd64",
-	Expiry:       "30d",
-	Name:         "{{ image.Distribution|lower }}-{{ image.Release }}-{{ image.Architecture }}-{{ image.Serial }}",
-	Serial:       "testing",
+var lxdDef = shared.Definition{
+	Image: shared.DefinitionImage{
+		Description:  "{{ image. distribution|capfirst }} {{ image.release }}",
```

Comment:
The space between "image." and "distribution" looks wrong. I guess it doesn't matter since it was broken before, but probably worth fixing :)

Reply:
Interestingly, this doesn't fail since pongo appears to still outputs the correct value. I'll fix it anyway since it looks weird.